.buildkite/pipeline.yml

```diff
@@ -40,21 +41,44 @@ steps:
         bats ./scripts/test.bats
 
   - label: test
-    branches: "!autodoc/master"
+    if: *not_scheduled_not_autodoc
     commands:
     - nix-build ci.nix -A packages.globacap.tests.globacap-test
     - ./result/bin/globacap-test
 
+  - label: nettest-local-chain
+    if: *not_scheduled_not_autodoc
+    env:
+      TEZOS_CLIENT_UNSAFE_DISABLE_DISCLAIMER: "Y"
+      NETTEST_NODE_ADDR: "localhost"
+      NETTEST_NODE_PORT: "8734"
+      MONEYBAG: "unencrypted:edsk47wWUSCHRDC1Hxtg7yxXzocwDjBJJRqTKnoP7htAifSpzwkg8K"
```

Comment:
Hm, I see that the same keys are used in stablecoin. It sounds fragile because we may get "wrong counter" errors. For local-chain it's less likely because blocks are produced very often, but I guess it can happen anyway if two tests are launched in parallel. Also the same problem can happen if two different PRs run nettest concurrently :thinking: 

Well, for CI in PR it's not a big problem, one can restart and hopefully such things will be rare. But for scheduled jobs it seems more likely and it's a bit more annoying because we will receive a notification (and we want to minimize number of such notifications so that they are rare and important). So I think we should use different moneybags in scheduled jobs in different projects.

For local-chain it's less important, but since we need it for testnet I think it's not a big problem to do it for local-chain as well. Is `edsk47wWUSCHRDC1Hxtg7yxXzocwDjBJJRqTKnoP7htAifSpzwkg8K` (or the corresponding address) hardcoded somewhere in the settings of local-chain? Can we hardcode a pool of addresses there (e. g. 10-15)? So that if I need to add nettest running in `tezos-btc` I will open this pool, pick a free address, mark it as used and will use it in `tezos-btc`.

Reply:
Well, buildkite jobs are always sequential, but yes it's probably a good idea to use different moneybags for different scheduled builds

Reply:
> Well, buildkite jobs are always sequential

AFAIU it may be wrong if we add more agents at some point. Or we may switch to something other than buildkite. But I guess it's not something we should care about now. Also we need to use different address for local-chain in buildkite and on gitlab. And we have two buildkite orgs (public and private).

Reply:
I've asked the ops what is the best way to distribute multiple bootstrap accounts between projects

Reply:
I've extended `boostrap_accounts` pool https://github.com/serokell/serokell-profiles/pull/486
Let's :zhdun: for it